<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTcpServerDemoClass</class>
 <widget class="QMainWindow" name="QTcpServerDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTcpServerDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listMsg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>581</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listClient">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>581</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelUI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTcpServerDemo.qrc"/>
 </resources>
 <connections/>
</ui>
